cortex/viz.py

```diff
@@ -14,18 +14,21 @@
 limitations under the License.
 """
 
-import matplotlib.pyplot as plt
```

Comment:
Need to do some large upgrades on these libraries ...  need to figure out testing ...